src/aggregator/server/m3msg/server.go

```diff
@@ -52,12 +55,19 @@ func NewServer(
 	s := &server{
 		aggregator: aggregator,
 		logger:     opts.InstrumentOptions().Logger(),
+		consumeMetrics: consumeMetrics{
+			handleLatency: opts.InstrumentOptions().MetricsScope().SubScope("consumer").Timer("handle-latency"),
```

Comment:
Can we use a timer that is set by the underlying timer options? You have to use `instrument.NewTimer` for this:
```go
scope := opts.InstrumentOptions().MetricsScope().SubScope("consumer")
instrument.NewTimer(scope, "handle-latency", opts.InstrumentOptions())
```